src/cml.js

```diff
@@ -330,11 +331,21 @@ class CML {
   }
 
   async ci(opts = {}) {
-    const { userEmail = GIT_USER_EMAIL, userName = GIT_USER_NAME } = opts;
+    const {
+      unshallow = false,
+      userEmail = GIT_USER_EMAIL,
+      userName = GIT_USER_NAME
+    } = opts;
 
     const driver = getDriver(this);
     const command = await driver.updateGitConfig({ userName, userEmail });
     await exec(command);
+    if (unshallow) {
+      const gitdir = await exec('git rev-parse --git-dir');
+      if (await fse.pathExists(gitdir + '/shallow')) {
```

Comment:
Safe to assume Git 2.15+ and use `git rev-parse --is-shallow-repository` instead?

Reply:
2.15 is EOL so probably safe :)

Reply:
feel free to push a commit @0x2b3bfa0